integration_tests/samples/socket_mode/bolt_adapter/aiohttp.py

```diff
@@ -0,0 +1,107 @@
+# ------------------
+# Only for running this script here
+import sys
+from os.path import dirname
+
+sys.path.insert(1, f"{dirname(__file__)}/../../../..")
+# ------------------
+
+import json
+import logging
+import os
+import asyncio
+from typing import Optional
+from time import time
+from slack_sdk.socket_mode.response import SocketModeResponse
+from slack_sdk.socket_mode.request import SocketModeRequest
+from slack_sdk.socket_mode.aiohttp import SocketModeClient
+
+from slack_bolt import App
+from slack_bolt.request import BoltRequest
+from slack_bolt.response import BoltResponse
+
+
+class SocketModeAdapter:
```

Comment:
eventually, this class will be placed under `slack_bolt.adapter`. that's why I named this as an adapter